### PR TITLE
ENH: Add new cluster for cloud chatops

### PR DESCRIPTION
**Pull request by khalford**
### Description:

Deploy a new cluster for chatops to make it easier for development changes

---

### Submitter:

Have you:

* [x] Labelled this PR, e.g. `bug`, `deployment`, `enhancement` ...etc.

- A `deployment` can be reviewed, and merged, by a single reviewer.
- It can only be used to deploy, change, or remove clusters based on existing patterns for staging.
- Anything involving prod, or production facing services must use the normal 2 person review.
- All other PR types require the usual PR process (e.g. 2 person).

---

### Reviewer

Have you:

* [ ] Verified this PR uses the correct label(s) based on the rules above?
* [ ] Checked if this could affect production (e.g. a global value that's changed without an override)?
* [ ] Tested setting this up, if it's not a deployment, to verify it can be redeployed with any documentation if appropriate?

### CODE REVIEW

File: clusters/dev/cloud-chatops/infra-values.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  addons:
+    ingress:
+      enabled: false
+      nginx:
+        release:
+          values:
+            controller:
+              service:
+                # create a floatip for ingress on your project and put it here
+                loadBalancerIP: "130.xxx.yyy.zzz"
```
Comment (anish-mudaraddi) [inline clusters/dev/cloud-chatops/infra-values.yaml:27]: just go ahead and delete this lines please if you're not enabling ingress

**Review by anish-mudaraddi — APPROVED** [review]
I'm assuming you're going to create a separate PR for adding `apps.yaml` and your chatops chart/secret to this cluster?

**Comment by anish-mudaraddi** [comment]
also I have dev management cluster on a feature branch to test alertmanager changes - once this PR is merged I'll rebase that feature branch onto main so your cluster can spin up

**Comment by khalford** [comment]
> I'm assuming you're going to create a separate PR for adding `apps.yaml` and your chatops chart/secret to this cluster?

I'm assuming so too. Just following the documentation :)

**Pull request merged**